bin/email_alert_service

```diff
@@ -28,7 +28,7 @@ begin
   listener.listen
 rescue SignalException => e
   logger.info "Received #{e}: exiting"
-rescue Exception => e
+rescue Exception => e # rubocop:disable Lint/RescueException
```

Comment:
Should we change these to `StandardError` rather than ignoring them?

Reply:
I'm trying to work out what we're throwing that we're catching here...

Reply:
@gpeng If I change this to `StandardError` will that still work?

Reply:
There's a comment in `listener.rb`: "Rescue any exception, not just StandardError and subclasses." Not sure of the context